src/app/pages/property-details/confirmation-modal/confirmation-modal.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="modal-container">
+  <div class="modal_card">
+    <div class="modal_header">
+      <span class="close_btn" (click)="closedModal()">
+        <i class="far fa-times-circle delete_icon fa-2x text_red"></i></span>
+      <span>
+        <h3>Delete</h3>
+      </span>
+    </div>
+    <div class="modal_body">Are you sure you want to delete this property?</div>
+    <div class="modal_footer">
+      <span>
+        <button class="btn btn-cancel" (click)="closedModal()">Cancel</button>
```

Comment:
Good work @MBenedicte, I think we can name this method better as `closeModal` as method names should tell what the behavior should be.